chainable/__init__.py

```diff
@@ -0,0 +1,46 @@
+from functools import reduce, _initial_missing
+from typing import TypeVar, Callable, overload, Generic
+from collections.abc import Iterable, Iterator
+
+__all__ = ["Chainable"]
```

Comment:
I know that almost the entire stdlib uses lists for `__all__`—but let's keep it a tuple if it's not about to change.